deps/rabbit/test/rabbit_stream_queue_SUITE.erl

```diff
@@ -2439,6 +2446,7 @@ max_segment_size_bytes_policy_validation(Config) ->
     ok.
 
 max_segment_size_bytes_policy(Config) ->
+    rabbit_ct_broker_helpers:enable_feature_flag(Config, stream_update_config_command),
```

Comment:
Same as above.

Reply:
I will remove these as the flag is enabled in in init_per_group